StatBot/StatBot/Modules/RiotModule.cs

```diff
@@ -1,19 +1,30 @@
 ﻿using Discord.Commands;
+using StatBot.Services;
 using System.Threading.Tasks;
 
 namespace StatBot.Modules
-{    
+{
     public class RiotModule : ModuleBase<SocketCommandContext>
     {
+        public ConfigService ConfigService { get; set; }
+
         /// <summary>
         /// Echo's back the string after the command.
         /// </summary>
-        /// <param name="echo">The string to echo.</param>
-        /// <returns></returns>
+        /// <param name="echo">The string to echo.</param>        
         [Command("echo")]
         public Task EchoAsync([Remainder] string echo)
-        {
+        {            
             return ReplyAsync(echo);
         }
+
+        /// <summary>
+        /// States the command prefix.
+        /// </summary>        
+        [Command("prefix")]
```

Comment:
I think there's actually an official way to provide a summary for a discord command in discord.NET

`[Summary("States the command prefix.")]`

This way when you run the (maybe builtin) `help` command, it can automatically yoink these.